lib/srv/regular/sshserver_test.go

```diff
@@ -1492,13 +1539,17 @@ func TestLimiter(t *testing.T) {
 	require.Error(t, err)
 
 	require.NoError(t, se.Close())
+	se.Wait()
 	require.NoError(t, clt.Close())
-	time.Sleep(50 * time.Millisecond)
+	require.ErrorIs(t, clt.Wait(), net.ErrClosed)
```

Comment:
I wanted to suggest to use `Wait()` on the client also. If we use it, do we still need time/sleeps here at all?

Upon closer look it doesn't seem like this test depends on timing at all so replacing it with fake clock basically does nothing here.

Reply:
Ah you are right, I'll remove then